### PR TITLE
Add Rainbow Serial Bumper for Trident

### PR DESCRIPTION
**Pull request by MapleLeafMakers**
### Which mods are added by this PR?

Rainbow Serial Bumper for Voron Trident

### How was it tested?

It's installed on my trident and functioning.

### Any background context you want to provide?

"Rainbow" alludes to the RGB Nature of the mod, and "Bumper" is like a vehicle bumper... 

### ⚠️⚠️**PLEASE READ AND COMPLETE THE CHECKLIST BELOW**⚠️⚠️

  * [x] I have read the rules available [here](https://github.com/VoronDesign/VoronUsers/wiki/Mod-Submission-Rules) and
        my mod adheres to these rules.
  * [x] This mod was created by myself and I agree to publish it under the repository
        [license](https://github.com/VoronDesign/VoronUsers/blob/master/LICENSE.md)

### CODE REVIEW

**Comment by mjonuschat** [comment]
@MapleLeafMakers Your PR is missing a `.metadata.yml` file. Please see these instructions on how to create one and add it to your commit: https://github.com/VoronDesign/VoronUsers/wiki/How-to-build-a-metadata.yml-file

**Comment by mjonuschat** [comment]
STL Orientation looks ok based on README and the fact that the insert will have the text embossed which the automated check isn’t aware off.

**Review by mjonuschat — APPROVED** [review]
STL Orientation looks ok based on README and the fact that the insert will have the text embossed which the automated check isn’t aware off.

**Comment by jontek2** [comment]
@MapleLeafMakers Seems like there's an extra image in the the images folder called "thumbnail.png" that's not used in the README. Could it be deleted or does it have a purpose?

**Comment by MapleLeafMakers** [comment]
> STL Orientation looks ok based on README and the fact that the insert will have the text embossed which the automated check isn’t aware off.

Can Confirm: 

`[oc]_insert.stl` is meant to have text engraved into the top surface, and then be flipped in the slicer to the orientation the bot suggested.

`[a]_main_body.stl` is orented that way purely for aesthetics, it looks awful printed on its back.

**Review by jontek2 — CHANGES_REQUESTED** [review]
This image called "thumbnail.png" listed in the image-folder but not being used in the README. Could it be deleted or does it have a purpose?

**Review by jontek2 — APPROVED** [review]
LGTM!

**Review by jontek2 — APPROVED** [review]

**Review by mjonuschat — COMMENTED** [review]
Did you remove the install.gif on purpose? Wasn’t it supposed to be the thumbnail.png?

File: printer_mods/MapleLeafMakers/Rainbow_Serial_Bumper/.metadata.yml
Comment (mjonuschat) [inline printer_mods/MapleLeafMakers/Rainbow_Serial_Bumper/.metadata.yml:1]: I think this was supposed to remove the thumbnail.png

Comment (MapleLeafMakers) [reply printer_mods/MapleLeafMakers/Rainbow_Serial_Bumper/.metadata.yml:1]: Maybe i was just testing you guys, or maybe I'm not paying enough attention, who's to say?

Comment (MapleLeafMakers) [reply printer_mods/MapleLeafMakers/Rainbow_Serial_Bumper/.metadata.yml:1]: @mjonuschat fixed, sorry about the mixup!

**Review by mjonuschat — APPROVED** [review]

**Pull request merged**